alembic/versions/2021_11_02_14_00_00-7eec82634342.py

```diff
@@ -0,0 +1,91 @@
+"""Add ondelete='CASCADE' to foreign keys that need it
+
+Revision ID: 7eec82634342
+Revises: 73c9d707134b
+Create Date: 2021-11-02 14:00:00
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7eec82634342'
+down_revision = '3fb8a6e2e0a5'
+
+from alembic import op
+
+
+def upgrade() -> None:
+    op.drop_constraint('downloadevent_version_id_fkey', 'downloadevent', type_='foreignkey')
```

Comment:
Would it make sense to try to delete old records that weren't deleted in the past and now have invalid or missing values?